README.md

```diff
@@ -38,7 +38,7 @@ $ brew install xhyve
 ```
 $ git clone https://github.com/ailispaw/boot2docker-xhyve
 $ cd boot2docker-xhyve
-$ make
+$ make    # You may be asked for your sudo password
```

Comment:
OK, I wasn’t sure and hadn’t waited for my sudo timeout to expire :)

On Tue, Oct 6, 2015 at 11:08 AM, A.I. <notifications@github.com> wrote:

>> @@ -38,7 +38,7 @@ $ brew install xhyve
>>  ```
>>  $ git clone https://github.com/ailispaw/boot2docker-xhyve
>>  $ cd boot2docker-xhyve
>> -$ make
>> +$ make    # You may be asked for your sudo password
> I believe you are asked in the only case of `make run`.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ailispaw/boot2docker-xhyve/pull/19/files#r41276636